processors/arxiv/docling_extractor.py

```
@@ -336,6 +337,13 @@ def process_pdf(self, arxiv_id: str, pdf_path: str,
                 }
             )
             
+            # Add markdown content and sections to the result
+            result['markdown'] = enhanced_markdown
+            result['sections'] = output_data.get('sections', [])
+            result['equations'] = [{'text': eq.text, 'page': eq.page} for eq in equations]
+            
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)